benchmark/gnmi/suite.go

```diff
@@ -37,52 +34,12 @@ type BenchmarkSuite struct {
 	value     input.Source
 }
 
-const onosComponentName = "onos-config"
-const testName = "gnmi-benchmark"
-
 // SetupSuite :: benchmark
 func (s *BenchmarkSuite) SetupSuite(c *benchmark.Context) error {
-	// Setup the Atomix controller
-	err := helm.Chart(onostest.ControllerChartName, onostest.AtomixChartRepo).
-		Release(onostest.AtomixName(testName, onosComponentName)).
-		Set("scope", "Namespace").
-		Install(true)
-	if err != nil {
-		return err
-	}
-
-	err = helm.Chart(onostest.RaftStorageControllerChartName, onostest.AtomixChartRepo).
-		Release(onostest.RaftReleaseName(onosComponentName)).
-		Set("scope", "Namespace").
-		Install(true)
-	if err != nil {
-		return err
-	}
-
-	// Install the onos-topo chart
-	err = helm.
-		Chart("onos-topo", onostest.AtomixChartRepo).
-		Release("onos-topo").
-		Set("replicaCount", 2).
-		Set("storage.controller", onostest.AtomixController(testName, onosComponentName)).
-		Set("image.tag", "latest").
-		Install(false)
-	if err != nil {
-		return err
-	}
-
-	// Install the onos-config chart
-	err = helm.
-		Chart("onos-config", onostest.OnosChartRepo).
-		Release("onos-config").
-		Set("replicaCount", 2).
-		Set("storage.controller", onostest.AtomixController(testName, onosComponentName)).
-		Set("image.tag", "latest").
-		Install(true)
-	if err != nil {
-		return err
-	}
-	return nil
+	umbrella := charts.CreateUmbrellaRelease().
+		Set("onos-topo.replicaCount", 2).
+		Set("onos-config.replicaCount", 2)
```

Comment:
@adibrastegarnia this should be 1 replica, right?

Reply:
The previous code set it to two. Not sure why.

Reply:
I think that I was just being optimistic 😛  If the tests pass we should leave it there though.